ci/kind/test-e2e-kind.sh

```diff
@@ -144,8 +144,8 @@ case $key in
     extra_vlan=true
     shift
     ;;
-    --deploy-external-server)
-    deploy_external_server=true
+    --deploy-no-antrea-worker)
```

Comment:
Can we use `deploy-no-antrea-worker` to replace `deploy-external-server`? I am not intending to achieve this in this PR, just want to know if `deploy-no-antrea-worker` can cover the test case of `deploy-external-server`. If so, given that `--deploy-external-server` is always added in `setup_cluster`, maybe we can consider fully replacing it with `deploy-no-antrea-worker` in the future.

Reply:
I think we can deploy host-network Pods we need in e2e tests on a Node without Antrea, but I'm not very sure if 
we can use `deploy-no-antrea-worker` to replace `deploy-external-server`.  @tnqn could you help confirm that in case we missed something?

Reply:
We need an IP that is not Node IP, so it can't replace it.